### ISSUE TITLE
ng-module-template missing carriage return

### ISSUE THREAD

**Issue by cgatian** [BeastCode/VSCode-Angular-TypeScript-Snippets#7]
IMO the declarations array should be on a new line to line up with all other properties.

The snippet currently produces:
```
@NgModule({ declarations: [],
  imports: [],
  exports: [],
  providers: [],
  bootstrap: []
})
```

**Comment by BeastCode** [comment]
Fixed in v2.4.6

**Comment by cgatian** [comment]
@BeastCode Thanks! Love these extensions!
